layouts/_default/list.html

```diff
@@ -29,9 +29,14 @@
 
 {{ define "main" }}
 
+{{ $truncate := default true .Site.Params.truncate }}
```

Comment:
I suggest using a construct like `$.Param "truncate"`—similar to what we do with `sharingicons`—so that the parameter can be set globally as well as on individual frontmatter. Hugo will use the `truncate` param that is most specific in that case, ie enabled globally but disabled on one particular post.

See: https://github.com/alanorth/hugo-theme-bootstrap4-blog/blob/master/layouts/_default/single.html#L77